7.1.0 (#395)
MetaMask/eth-sig-util: CHANGELOG.md

# Changelog

All notable changes to this project will be documented in this file.

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [Unreleased]

## [7.1.0]

### Changed

> - Bypass numerical address cast ([#391](https://github.com/MetaMask/eth-sig-util/pull/391))

Reviewer: ```suggestion
- **BREAKING**: Values of type `number` are not accepted as address parameter anymore. Valid values are `string` and `Uint8Array`. ([#391](https://github.com/MetaMask/eth-sig-util/pull/391))
```
@OGPoyraz I think the change is breaking? Please confirm if this is right.

Reviewer 2: You're right this message is not clear. [The removed part on this PR](https://github.com/MetaMask/eth-sig-util/pull/391) is removing the cast of the numerical address cast, I've adjusted the PR. 
I am a bit unsure that it should be a breaking change or not, what do you think?

Reviewer: I would say it's breaking because:
- There exists input that was previously interpreted as valid and handled, which will now break
  - We can see example of this because of the need to remove these test cases: https://github.com/MetaMask/eth-sig-util/pull/391/files#diff-f6dbf9fbb07112db7776d1c764cea2d7e6df9d05cbdd800178d56ef5215a807bL4300-L4301

Reviewer 2: Thanks @legobeat !
I've just spin up another release here, so closing this.

> - Bump the npm_and_yarn group across 1 directory with 2 updates ([#374](https://github.com/MetaMask/eth-sig-util/pull/374))

Reviewer: ```suggestion
```
`devDependency` bumps in general not relevant for changelog

> - Bump @metamask/utils from 9.0.0 to 9.3.0 ([#392](https://github.com/MetaMask/eth-sig-util/pull/392))

Reviewer: ```suggestion
- Bump `@metamask/utils` from `^9.0.0` to `^9.3.0` ([#392](https://github.com/MetaMask/eth-sig-util/pull/392))
```

> - Drop support for Node.js versions 16, 21 ([#390](https://github.com/MetaMask/eth-sig-util/pull/390))

Reviewer: ```suggestion
- **BREAKING**: Drop support for Node.js versions 16, 21 ([#390](https://github.com/MetaMask/eth-sig-util/pull/390))
```
